### PR TITLE
fix: mock for firefly

### PR DESCRIPTION
**Pull request by UncleBill**
MF-0000

### CODE REVIEW

File: packages/flags/src/flags/buildInfo.ts [hunk 1]
```diff
@@ -28,7 +28,9 @@ export async function getBuildInfo(): Promise<BuildInfoFile> {
         }
     }
 }
-export let env: BuildInfoFile
+export let env: BuildInfoFile = {
+    channel: 'beta',
```
Comment (Jack-Works) [inline packages/flags/src/flags/buildInfo.ts]:
```suggestion
    channel: 'stable',
```
?

and access to this variable before call setupBuildInfo should be a fatal error.

File: packages/mask/package.json [hunk 2]
```diff
@@ -29,7 +29,10 @@
     }
   },
   "exports": {
-    ".": null
+      "./crypto": {
+          "mask-src": "./background/services/crypto/index.ts",
+          "default": "./background/services/crypto/index.js"
+      }
```
Comment (Jack-Works) [inline packages/mask/package.json]: this package should not have any exports. can you explain why you need this?

Comment (UncleBill) [reply packages/mask/package.json]: We would decode encrypted post payload directly in firefly.

Comment (Jack-Works) [reply packages/mask/package.json]: you should use `@masknet/encryption` package directly, which is published on npm. you can see how `packages/app` uses that package as a reference.

File: packages/flags/src/flags/buildInfo.ts [hunk 3]
```diff
@@ -28,7 +28,9 @@ export async function getBuildInfo(): Promise<BuildInfoFile> {
         }
     }
 }
-export let env: BuildInfoFile
+export let env: BuildInfoFile = {
```
Comment (Jack-Works) [inline packages/flags/src/flags/buildInfo.ts]: access to this variable before calling setupBuildInfo should be a fatal error, so I'd rather leave it `undefined`.

**Pull request closed without merging**